core/modules/installer/installer.browser.pages.inc

```diff
@@ -0,0 +1,884 @@
+        }
+      }
+    }
+
+    return backdrop_get_form('installer_browser_installation_select_versions_form', $missing_projects);
+  }
+  else {
+    backdrop_goto('admin/modules/installer-browser/install/enable');
+  }
+}
+
+/**
+ * Task page for the Enable projects installation task.
+ *
+ * Show a form which lets the user enable the newly installed projects.
+ */
+function installer_browser_installation_enable_page($type = 'enable') {
+  module_load_include('inc', 'installer', 'installer.browser');
+  $installed_projects = installer_browser_get_installed_projects();
+
+  if (count($installed_projects) > 0) {
+    return backdrop_get_form('installer_browser_installation_' . $type . '_form', $installed_projects);
+  }
+  else {
+    backdrop_goto('admin/modules/installer-browser');
+  }
+}
+
+/**
+ * Form builder for theme install.
+ *
+ * @param $projects
+ *   An array of projects selected for install, keyed by project name.
+ */
+function installer_browser_installation_theme_form($form, &$form_state, $projects) {
+  module_load_include('inc', 'installer', 'installer.browser');
+  $form['instructions'] = array(
+  '#type' => 'item',
+  '#markup' => t('The themes you selected have been successfully installed.
+    You may enable installed themes here or on the main !themes page. ', array('!themes' => l(t('Themes'), 'admin/appearance')))
+  );
+
+  $theme_default = config_get('system.core', 'theme_default');
+  $themes = system_rebuild_theme_data();
+
+  foreach ($projects as $project) {
+    if ($project['type'] == 'theme' && isset($themes[$project['name']])) {
+      $theme = $project['name'];
+      $form[$theme] = array(
+        '#type' => 'fieldset',
+        '#collapsed' => FALSE,
+        '#collapsible' => FALSE,
+        '#title' => $themes[$theme]->info['name'],
+        '#attributes' => array(
+          'class' => array('browser-installed-theme'),
+        ),
+      );
+      if (file_exists($themes[$theme]->info['screenshot'])) {
+        $form[$theme]['image'] = array(
+          '#type' => 'item',
+          '#markup' => theme('image', array('uri' => $themes[$theme]->info['screenshot'])),
+          '#attributes' => array(
+            'class' => array('browser-installed-theme-image'),
+          ),
+        );
+      }
+      if ($theme == $theme_default) {
+        $form[$theme]['is_default'] = array(
+          '#type' => 'markup',
+          '#markup' => t('Default theme'),
+        );
+      }
+      else {
+        $form[$theme]['set'] = array(
+          '#type' => 'submit',
+          '#value' => 'Set as default theme',
+          '#name' => $theme . '-submit',
+          '#submit' => array('installer_browser_installation_theme_set'),
+        );
+      }
+    }
+  }
+
+  if (in_array('module', installer_browser_get_installed_types())) {
+    $form['next_link'] = array(
+      '#type' => 'link',
+      '#title' => 'Next: Install downloaded modules',
```

Comment:
This text should read: `Next: enable new modules`.
* we are not downloading
* we are enabling, not installing :)